src/pages/Home.js

```diff
@@ -2,20 +2,20 @@ import styled from 'styled-components';
 import Header from '../components/Header';
 import Button from '../components/Button';
 
-export default function Home({ handleCheckIsAdult }) {
+export default function Home({ onCheckIsAdult }) {
   return (
```

Comment:
so here (and in line 15 and 18) you would have to change it again, but it would be the "real name" of your function with `handleCheckIsAdult`, that you then call with the onClick-Event